orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/instance/AbstractInstancesCheckTask.groovy

```diff
@@ -119,7 +126,16 @@ abstract class AbstractInstancesCheckTask extends AbstractCloudProviderAwareTask
         }
       }
 
-      verifyServerGroupsExist(stage)
+      try {
+        verifyServerGroupsExist(stage)
+      } catch (MissingServerGroupException e) {
+        if (waitForUpServerGroup()) {
```

Comment:
Yeah makes sense, should the timeout for the whole abstract task be changed? Or just wait for up instances? Or do are theses classes instantiated once per stage, and keep their state through the execution allowing me to timeout only when the server group is missing?